.github/workflows/build.yml

```diff
@@ -29,27 +29,35 @@ jobs:
             matrix:
                 php: ["8.1", "8.2"]
```

Comment:
In the InvoicingPlugin (https://github.com/Sylius/InvoicingPlugin/blob/1.0/.github/workflows/build.yml#L22), we kept only PHP 8.3 in the builds, as well as we did in the PayPalPlugin (https://github.com/Sylius/PayPalPlugin/blob/1.7/.github/workflows/build.yml#L22)